hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1236,10 +1236,16 @@
       name: chip_sw_alert_handler_lpg
       desc: '''Verify alert's low_power_group(lpg).
 
-            Choose an always-on block and send alerts during deep sleep and normal sleep mode,
+            Three methods to check LPG:
+            - Choose an always-on block and send alerts during deep sleep and normal sleep mode,
```

Comment:
@cindychip we may also want to pick a non-always on block and check that against normal sleep mode, since those blocks will not get reset.

Reply:
> @cindychip we may also want to pick a non-always on block and check that against normal sleep mode, since those blocks will not get reset.

In that case, alert_handler programmed s.t.
escalation stays at phase1 to avoid reset.